Not running on ubuntu 20.04

Fresh installed ubuntu 20.04

```bash
	docker run -it \
		--rm \
		--env="DISPLAY" \
		--volume="${XAUTHORITY:-${HOME}/.Xauthority}:/root/.Xauthority:ro" \
		--volume="/tmp/.X11-unix:/tmp/.X11-unix" \
		--volume="winehome:/home/wineuser" \
		--hostname="$(hostname)" \
		--name="wine" \
		scottyhardy/docker-wine wine notepad
```

result: 

```bash
docker run -it \
        --rm \
        --env="DISPLAY" \
        --volume=":/root/.Xauthority:ro" \
        --volume="/tmp/.X11-unix:/tmp/.X11-unix" \
        --volume="winehome:/home/wineuser" \
        --hostname="" \
        --name="wine" \
        scottyhardy/docker-wine wine notepad
No protocol specified
0010:err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
0010:err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
0010:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80004002
0010:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80004002
0010:err:ole:get_local_server_stream Failed: 80004002
0009:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0009:err:winediag:nodrv_CreateWindow Make sure that your X server is running and that $DISPLAY is set correctly.
make: *** [Makefile:5: run] Fehler 1
```

--- a/docker-wine
+++ b/docker-wine
@@ -276,7 +276,7 @@ USE_RDP_SERVER="no"
 USER_HOME="/home/wineuser"
 USER_VOLUME="winehome"
 WORKDIR="${USER_HOME}"
-XVFB="no"
+USE_XVFB="no"
 XVFB_RESOLUTION="320x240x8"
 XVFB_SCREEN="0"
 XVFB_SERVER=":95"
@@ -328,10 +328,10 @@ while [ $# -gt 0 ]; do
         SHM_SIZE="${1#*=}"
         ;;
     --xvfb)
-        XVFB="yes"
+        USE_XVFB="yes"
         ;;
     --xvfb=*)
-        XVFB="yes"
+        USE_XVFB="yes"
         IFS=, read -r XVFB_SERVER XVFB_SCREEN XVFB_RESOLUTION <<< "${1#*=}"
         ;;
     --home-volume=*)
@@ -406,8 +406,8 @@ if [ "${USE_RDP_SERVER}" != "no" ] && [ "${NO_RDP}" != "no" ]; then
 fi
 
 # Run xvfb and send everything into the void
-if [ "${XVFB}" == "yes" ] ; then
-    add_run_arg --env="XVFB=yes"
+if [ "${USE_XVFB}" == "yes" ] ; then
+    add_run_arg --env="USE_XVFB=yes"
     add_run_arg --env="XVFB_SERVER=${XVFB_SERVER}"
     add_run_arg --env="XVFB_SCREEN=${XVFB_SCREEN}"
     add_run_arg --env="XVFB_RESOLUTION=${XVFB_RESOLUTION}"
@@ -472,7 +472,7 @@ else
     elif [ "$(uname)" == "Linux" ]; then
 
         # Check for .Xauthority which is required for authenticating as the current user on the host's X11 server
-        if [ -z "${XAUTHORITY:-${HOME}/.Xauthority}" ] && [ "${XVFB}" == "no" ]; then
+        if [ -z "${XAUTHORITY:-${HOME}/.Xauthority}" ] && [ "${USE_XVFB}" == "no" ]; then
             echo "ERROR: No valid .Xauthority file found for X11"
             exit 1
         fi

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+is_enabled () {
+    echo "$1" | grep -q -i -E "^(yes|on|true|1)$"
+}
+
+is_disabled () {
+    echo "$1" | grep -q -i -E "^(no|off|false|0)$"
+}
+
+
 # Set user account and run values
 USER_NAME=${USER_NAME:-wineuser}
 USER_UID=${USER_UID:-1010}
@@ -10,7 +19,7 @@ RDP_SERVER=${RDP_SERVER:-no}
 RUN_AS_ROOT=${RUN_AS_ROOT:-no}
 FORCED_OWNERSHIP=${FORCED_OWNERSHIP:-no}
 TZ=${TZ:-UTC}
-XVFB=${XVFB:-no}
+USE_XVFB=${USE_XVFB:-no}
 
 # Create the user account
 ! grep -q ":${USER_GID}:$" /etc/group && groupadd --gid "${USER_GID}" "${USER_NAME}"
@@ -22,7 +31,7 @@ useradd --shell /bin/bash --uid "${USER_UID}" --gid "${USER_GID}" --password "${
 # Take ownership of user's home directory if owned by root or if FORCED_OWNERSHIP is enabled
 OWNER_IDS="$(stat -c "%u:%g" "${USER_HOME}")"
 if [ "${OWNER_IDS}" != "${USER_UID}:${USER_GID}" ]; then
-    if [ "${OWNER_IDS}" == "0:0" ] || [ "${FORCED_OWNERSHIP}" == "yes" ]; then
+    if [ "${OWNER_IDS}" == "0:0" ] || is_enabled "${FORCED_OWNERSHIP}"; then
         chown -R "${USER_UID}":"${USER_GID}" "${USER_HOME}"
     else
         echo "ERROR: User's home '${USER_HOME}' is currently owned by $(stat -c "%U:%G" "${USER_HOME}")"
@@ -35,21 +44,21 @@ fi
 ln -snf "/usr/share/zoneinfo/${TZ}" /etc/localtime && echo "${TZ}" > /etc/timezone
 
 # Run in X11 redirection mode (default) or with xvfb
-if echo "${RDP_SERVER}" | grep -q -i -E "^(no|off|false|0)$"; then
+if echo is_disabled "${RDP_SERVER}"; then
 
     # Set up pulseaudio for redirection to UNIX socket
     [ -f /root/pulse/client.conf ] && cp /root/pulse/client.conf /etc/pulse/client.conf
 
     # Run xvfb
-    if echo "${XVFB}" | grep -q -i -E "^(yes|on|true|1)$"; then
+    if is_enabled "${USE_XVFB}"; then
         nohup /usr/bin/Xvfb "${XVFB_SERVER}" -screen "${XVFB_SCREEN}" "${XVFB_RESOLUTION}" >/dev/null 2>&1 &
     fi
 
     # Run in X11 redirection mode as $USER_NAME (default)
-    if echo "${RUN_AS_ROOT}" | grep -q -i -E "^(no|off|false|0)$"; then
+    if is_disabled "${RUN_AS_ROOT}"; then
 
         # Copy and take ownership of .Xauthority for X11 redirection
-        if [ -f /root/.Xauthority ] && [ "${XVFB}" == "no" ]; then
+        if [ -f /root/.Xauthority ] && is_disabled "${USE_XVFB}"; then
             cp /root/.Xauthority "${USER_HOME}"
             chown "${USER_UID}":"${USER_GID}" "${USER_HOME}/.Xauthority"
         fi
@@ -58,7 +67,7 @@ if echo "${RDP_SERVER}" | grep -q -i -E "^(no|off|false|0)$"; then
         exec gosu "${USER_NAME}" "$@"
 
     # Run in X11 redirection mode as root
-    elif echo "${RUN_AS_ROOT}" | grep -q -i -E "^(yes|on|true|1)$"; then
+    elif is_enabled "${RUN_AS_ROOT}"; then
         exec "$@"
     else
         echo "ERROR: '${RUN_AS_ROOT}' is not a valid value for RUN_AS_ROOT"
@@ -66,7 +75,7 @@ if echo "${RDP_SERVER}" | grep -q -i -E "^(no|off|false|0)$"; then
     fi
 
 # Run in RDP server mode
-elif echo "${RDP_SERVER}" | grep -q -i -E "^(yes|on|true|1)$"; then
+elif is_enabled "${RDP_SERVER}"; then
 
     # Exit if using nordp image
     if ! [ -f /usr/sbin/xrdp ]; then
@@ -83,9 +92,9 @@ elif echo "${RDP_SERVER}" | grep -q -i -E "^(yes|on|true|1)$"; then
     else
         /usr/sbin/xrdp
 
-        if echo "${RUN_AS_ROOT}" | grep -q -i -E "^(no|off|false|0)$"; then
+        if is_disabled "${RUN_AS_ROOT}"; then
             exec gosu "${USER_NAME}" "$@"
-        elif echo "${RUN_AS_ROOT}" | grep -q -i -E "^(yes|on|true|1)$"; then
+        elif is_enabled "${RUN_AS_ROOT}"; then
             exec "$@"
         else
             echo "ERROR: '${RUN_AS_ROOT}' is not a valid value for RUN_AS_ROOT"

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -10,6 +10,7 @@ RDP_SERVER=${RDP_SERVER:-no}
 RUN_AS_ROOT=${RUN_AS_ROOT:-no}
 FORCED_OWNERSHIP=${FORCED_OWNERSHIP:-no}
 TZ=${TZ:-UTC}
+XVFB=${XVFB:-no}
 
 # Create the user account
 ! grep -q ":${USER_GID}:$" /etc/group && groupadd --gid "${USER_GID}" "${USER_NAME}"

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -44,7 +44,7 @@ fi
 ln -snf "/usr/share/zoneinfo/${TZ}" /etc/localtime && echo "${TZ}" > /etc/timezone
 
 # Run in X11 redirection mode (default) or with xvfb
-if echo is_disabled "${RDP_SERVER}"; then
+if is_disabled "${RDP_SERVER}"; then
 
     # Set up pulseaudio for redirection to UNIX socket
     [ -f /root/pulse/client.conf ] && cp /root/pulse/client.conf /etc/pulse/client.conf

